modules/layer1.js

```diff
@@ -0,0 +1,17 @@
+var Layer1 = `
+                    <img src="profile.jpg">
+                    <div id="progress">
+                        <span class="progress_count">
+                            <h3 id="post_count">4,259</h3>
+                            <p>Posts</p>
+                        </span>
+                        <span class="progress_count">
+                            <h3 id="followers_count">6.8M</h3>
+                            <p>Followers</p>
+                        </span>
+                        <span class="progress_count">
+                            <h3 id="follow_count">291</h3>
+                            <p>Following</p>
+                        </span>
+                    </div>`;
+export default Layer1;
```

Comment:
Let's add an empty line at the end of this file.

Reply:
okay sir